go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/pyroscope-io/otel-profiling-go v0.4.0
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.1
+	github.com/stretchr/testify v1.8.2
```

Comment:
this bump wasn't intentional and appears to have been auto applied by `go mod tidy` (?)

Reply:
Not a problem.